util/test/demos/demos.vcxproj

```diff
@@ -477,4 +478,7 @@
       <Visible>false</Visible>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="..\..\..\.clang-format" />
+  </ItemGroup>
```

Comment:
What's the reason for including the .clang-format file here?

Reply:
Good question:  I missed that.  Don't know.  I think it may have been me trying to get VS(2022) to adopt that formatting style (or me investigating whether VS would auto-format differently).

Have now removed it.